book/06-github/sections/2-contributing.asc

```diff
@@ -270,11 +270,11 @@ To https://github.com/tonychacon/blink
    ef4725c..3c8d735  slower-blink -> slow-blink
 ----
 
-<1> Add the original repository as a remote named ``upstream''
-<2> Fetch the newest work from that remote
-<3> Merge the main branch of that repository into your topic branch
-<4> Fix the conflict that occurred
-<5> Push back up to the same topic branch
+<1> Add the original repository as a remote named ``upstream''.
```

Comment:
Hmm. Somehow this doesn't quite sit well with me. Can we change the ` ``upstream'' ` here to monospace?

```suggestion
<1> Add the original repository as a remote named `upstream`.
```